tests/unit/windows/mocks/mockwinapilayer.cpp

```diff
@@ -0,0 +1,88 @@
+// local includes
+#include "mockwinapilayer.h"
+
+namespace {
```

Comment:
I am not familiar with gmock, but is there a way to parametrize these?

Reply:
I'm not sure I follow? I have defined these const's to be reusable within the first UT and future UTs.

Reply:
If you meant `TEST_P`, I have decided against it, since it requires it's own test fixture and the data set is just too small to justify it creating for a single function.